go.mod

```
@@ -37,6 +37,7 @@ require (
 	github.com/golang-jwt/jwt/v5 v5.2.2
 	github.com/google/go-containerregistry v0.20.3
 	github.com/google/go-github/v56 v56.0.0
+	github.com/google/go-github/v71 v71.0.0
```

```suggestion
	github.com/google/go-github/v71 v71.0.0
```

Need to update `internal/gitprovider/github/github.go` and `internal/gitprovider/github/github_test.go` to go along with this.